internal/commands/project.go

```diff
@@ -324,7 +269,12 @@ func runCreateProjectCommand(
 				return errors.Wrapf(err, "%s", failedCreatingProj)
 			}
 		}
-
+		if wrappers.FeatureFlags[accessManagementEnabled] {
```

Comment:
should it be in groups, no? then it doesn't matter if it called from create project/ update